spec/models/profile_spec.rb

```diff
@@ -404,10 +404,8 @@
       profile.deactivate_for_fraud_review
 
       expect(profile).to_not be_active
-      expect(profile.fraud_review_pending).to eq(true)
-      expect(profile.fraud_review_pending_at).to_not be_nil
-      expect(profile.fraud_rejection).to eq(false)
-      expect(profile.fraud_rejection_at).to be_nil
+      expect(profile.fraud_review_pending?)
+      expect(profile.fraud_rejection?)
```

Comment:
This syntax was surprising to me, so I tried it out. Rspec doesn't check the result at all, since `profile.fraud_rejection?` returns false and the test passes. This needs to change in all the other tests with similar syntax too.
```suggestion
      expect(profile.fraud_review_pending?).to eq(true)
      expect(profile.fraud_rejection?).to eq(false)
```

Reply:
Whoa good catch thanks!